unlock-app/src/components/interface/Launcher.tsx

```diff
@@ -0,0 +1,75 @@
+import Image from 'next/image'
+import Link from 'next/link'
+import { FiExternalLink as ExternalLinkIcon } from 'react-icons/fi'
+
+const options = [
+  {
+    image: {
+      src: '/images/illustrations/events/outmetaverse.svg',
+      width: 320,
+      height: 200,
+      alt: 'organize event',
+    },
+    cta: 'Organize an event and sell tickets',
+    href: '/event/new',
+  },
+  {
+    image: {
+      src: '/images/illustrations/certifications/img-handoffDoc.svg',
+      width: 250,
+      height: 200,
+      alt: 'create certification',
+    },
+    cta: 'Certify & Show the expertise on chain.',
+    href: '/certification/new',
+  },
+]
+
+export const Launcher = () => {
+  return (
+    <div className="flex flex-col items-center text-center">
+      <h1 className="text-3xl font-bold mb-2">
+        What do you want to achieve today?
+      </h1>
+      <h2 className="text-lg font-light text-center">
+        Let&apos;s begin by selecting a starting point that has easy and
+        <br />
+        manageable steps for you to follow! 😊
+      </h2>
+      <ul className="flex gap-4 mt-8 mb-12 flex-col md:flex-row">
+        {options.map(({ cta, image, href }) => (
+          <li
+            key={image.alt}
+            className="relative h-96 overflow-hidden bg-[#FFF7E8] w-96 rounded-lg drop-shadow-2xl"
```

Comment:
I will keep the shadow only with a hover
```suggestion
            className="relative h-96 overflow-hidden bg-[#FFF7E8] w-96 rounded-lg duration-200 hover:drop-shadow-2xl"
```

Reply:
Would add some smaller shadow too